rpcd/playbooks/roles/rpc_support/tasks/retrieve_auth_key.yml

```diff
@@ -13,6 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- name: Install pip packages for memecache module to work
```

Comment:
@prometheanfire: The rpc-support role will fail when trying to use the memcache ansible module if these pacakges are installed. It failed with this error message:
https://gist.github.com/alextricity25/3919818f49c94136df06c06185e0d92d
It seems to me that we are going to have to install the memcache python modules now in order to get it going.